packages/inter-protocol/src/vaultFactory/vaultKit.js

```diff
@@ -27,9 +27,9 @@ export const prepareVaultKit = baggage => {
         vault: holder.getSubscriber(),
       },
       invitationMakers: Far('invitation makers', {
-        AdjustBalances: holder.makeAdjustBalancesInvitation,
-        CloseVault: holder.makeCloseInvitation,
-        TransferVault: holder.makeTransferInvitation,
+        AdjustBalances: () => holder.makeAdjustBalancesInvitation(),
```

Comment:
The one test that exercised vault `invitationMakers` was is no longer used: https://github.com/Agoric/agoric-sdk/blob/166f839a1341e330a9a3b04871b9336af0ddd86e/packages/inter-protocol/test/vaultFactory/vault-contract-wrapper.js#L221

I'll remove that and cover it somewhere else.

Reply:
No tests of `invitationMakers` from VaultFactory. They all call the methods directly on the vault kit.

Since invitationMakers is mostly to serve continuing invitations and that's mostly used for smart wallet, I'm inclined to test that in the smart wallet integration. Right now that's this `agops-vaults-integration.sh` that doesn't run in CI.